pycds/orm/native_matviews/version_fecff1a73d7e.py

```diff
@@ -0,0 +1,98 @@
+from sqlalchemy.orm import declarative_base
+
+from sqlalchemy import (
+    Index,
+    Column,
+    Integer,
+    String,
+    ForeignKey,
+    func,
+    select,
+    text,
+    MetaData,
+)
+from sqlalchemy.dialects.postgresql import aggregate_order_by, ARRAY, TEXT
+
+from pycds.alembic.extensions.replaceable_objects import ReplaceableNativeMatview
+
+# These cannot be imported from `pycds` (top-level) because it is not yet fully
+# initialized at this point.
+from pycds.orm.tables import Variable
+from pycds.orm.native_matviews import VarsPerHistory
+from pycds.context import get_schema_name
+from pycds.orm.view_base import make_declarative_base
+from pycds.util import variable_tags
+
+Base = make_declarative_base()
+schema_name = get_schema_name()
+
+
+# This CTE is used in the selectable for matview `CollapsedVariables`. For each history,
+# it aggregates values computed from columns of `Variable`. `VarsPerHistory` provides
+# the connection between variables and histories. The main reason for its existence is
+# to enable the arrays of variable tags to be flattened and unique'd, which requires or
+# at least is most easily done with an intermediate stage created here. Other things are
+# placed here too, mainly for readability, but they could just as easily be computed
+# directly within the matview selectable.
+aggregated_vars = (
+    select(
+        VarsPerHistory.history_id.label("history_id"),
+        func.array_agg(VarsPerHistory.vars_id).label("vars_ids"),
+        func.array_agg(aggregate_order_by(variable_tags(Variable), Variable.id)).label(
```

Comment:
Unclear exactly what "all variable tags" is for as a new user, could be explained a little better.